common/changes/@itwin/breakdown-trees-react/tree_widget_x_2023-06-29-15-44.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/breakdown-trees-react",
+      "comment": "Update AppUI to 4.2.0",
```

Comment:
is this relevant anymore? it doesn't look like the package has changes in this pr

Reply:
Yes looks like this is outdated. Removed.